src/pods/toolbar/components/export-button/export-button.business.ts

```diff
@@ -28,10 +29,68 @@ export const calculateTableEndYPosition = (table: TableVm): number => {
   return table.y + tableHeight;
 };
 
+export const doesTableOverlap = (
+  tableA: TableVm,
+  tables: TableVm[]
+): boolean => {
+  if (tables.length === 0) {
+    return false;
+  }
+
+  const tablesMinusOriginTableCollection = tables.filter(
+    t => t.id !== tableA.id
+  );
+
+  return tablesMinusOriginTableCollection.some(tableB =>
+    doTablesOverlap(tableA, tableB)
+  );
+};
+
+export const placeTableWithoutOverlap = (
+  table: TableVm,
+  tables: TableVm[]
+): TableVm => {
+  let newTable = table;
+  let attempts = 0;
+
+  while (
+    doesTableOverlap(newTable, tables) &&
+    attempts < TABLE_CONST.MAX_PLACEMENT_ATTEMPTS
+  ) {
+    newTable = {
+      ...newTable,
+      x: newTable.x - TABLE_CONST.TABLE_SHIFT_DISTANCE,
+    };
+
+    if (newTable.x < 0) {
+      newTable = {
+        ...newTable,
+        x: TABLE_CONST.TABLE_SHIFT_DISTANCE,
```

Comment:
before it was x: 0. In fact, i have used x: TABLE_CONST.TABLE_SHIFT_DISTANCE. If we used x: 0 when we export the schema we lose the relations, with this solution we move to the right the table 30 px, and dont lose the relations when we export.